src/index.html

```diff
@@ -1,22 +1,36 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Moyo Header</title>
     <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"/>
+    <link rel="stylesheet" href="style.css"/>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="MOYO logo"/>
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="nav__list is-active"><a href="#">Apple</a></li>
+          <li class="nav__list"><a href="#">Samsung</a></li>
+          <li class="nav__list"><a href="#">Smartphones</a></li>
+          <li class="nav__list" data-qa="hover"><a href="#">Laptops & Computers</a></li>
+          <li class="nav__list"><a href="#">Gadgets</a></li>
+          <li class="nav__list"><a href="#">Tablets</a></li>
+          <li class="nav__list"><a href="#">Photo</a></li>
+          <li class="nav__list"><a href="#">Video</a></li>
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol and be followed by the identifier of the section you want to link to.